server/conf.js

```diff
@@ -23,8 +23,8 @@ const DEFAULTS = {
     }
   },
   auth: {
-    grantAdminToLocalRequest: false,
-    enableExtHeaderAccess: false
+        grantAdminToLocalRequest: true,
+        enableExtHeaderAccess: true
```

Comment:
NO!  Do not allow these settings in production!  You may enable them in development mode if you need.